receiver/main.py

```diff
@@ -0,0 +1,15 @@
+import read_input
+import cal_para_values 
+from cal_para_values  import temperature_lst , chargeRate_lst, soc_lst
+
+if __name__ == '__main__':
+
+    while True:
+        temperature,soc,chargeRate, validate_data = read_input.read_sender_input()

```

Comment:
If the code doesn't know how to handle invalid data (it just exits here), better to use an exception - that's what exceptions are meant for.